features/support/capybara.rb

```diff
@@ -95,29 +95,28 @@ def is_a?(klass)
 end
 
 Capybara.register_driver :headless_chrome do |app|
-  options = Selenium::WebDriver::Chrome::Options.new
+  options = Selenium::WebDriver::Options.chrome(
+    logging_prefs: { performance: 'ALL', browser: 'ALL' }
+  )
 
-  options.add_argument('--headless')
+  options.add_argument('--headless=new')
   options.add_argument('--no-sandbox')
   options.add_argument('--disable-popup-blocking')
   options.add_argument('--window-size=1280,2048')
   options.add_argument('--host-resolver-rules=MAP * ~NOTFOUND , EXCLUDE *localhost*')
   options.add_argument('--disable-search-engine-choice-screen')
+  options.add_argument('--disable-gpu')
```

Comment:
This fixes the timeouts, for some strange reason...

Reply:
[Headless options](https://github.com/GoogleChrome/chrome-launcher/blob/main/docs/chrome-flags-for-tools.md#headless):
*`--headless`: Run in headless mode, i.e., without a UI or display server dependencies.
*`--headless=new`: New, native Headless mode. ([previously](https://bugs.chromium.org/p/chromium/issues/detail?id=1380881), --headless=chrome)
[...]
*`--disable-gpu`: Was often [used](https://bugs.chromium.org/p/chromium/issues/detail?id=737678) along with --headless, but as of 2021, isn't needed.

Maybe it wasn't necessary for --headless but it is for  --headless=new?

Reply:
I don't know, this is weird...

The summary is:
- with `--headless` and **without** `--disable-gpu` - timeout
- with `--headless=new` and **without** `--disable-gpu` - timeout
- with `--headless=new` and with `disable-gpu` - works :heavy_check_mark:  

I don't know if `--headless` and `--disable-gpu` would work, but I guess we can keep `--headless=new` anyway.